tasks.py

```diff
@@ -124,6 +124,20 @@ def deploy(ctx, space=None, branch=None, login=None, yes=False):
         print("Check logs for more detail.")
         return sys.exit(1)
 
+    # Allow proxy to connect to eregs via internal route
+    add_network_policy = ctx.run('cf add-network-policy proxy eregs'.format(cmd, space),
+        echo=True,
+        warn=True
+    )
+    if not add_network_policy.ok:
+        print(
+            "Unable to add network policy. Make sure the proxy app is deployed.\n"
+            "For more information, check logs."
+        )
+
+        # Fail the build because eregs will be down until the proxy is can connect
```

Comment:
Suggested syntax:
>`# Fail the build because eregs will be down until the proxy can connect`

Reply:
😆 yes, will fix that, thanks!

Reply:
@jason-upchurch @pkfec fixed, thanks for catching that! Also put in this PR: https://github.com/fecgov/fec-cms/pull/4215